### DISCUSSION TITLE
[Enhancement] [table] table 支持控制x行省略，悬浮提示

### DISCUSSION THREAD

**Discussion by ghost** [element-plus/element-plus#16839] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

当单元格内容过长，可以自定义单元格元素几行后...，并鼠标提示，这可能会导致有的row会是一行，有的row会是3行 每一行的高度不一致，不知虚拟表格能否支持，这样可以提升渲染的速度不需要在单独在template里写个el-tooltip

### Additional comments

_(empty)_

**Comment by jinceon** [comment]
那怎样定义多少是一行呢？
假设width=200，要计算一个字占多少宽，多少个字就算一行？
